src/test/java/atf/CVSB_1865/TestingFacility_CVSB_1865.java

```diff
@@ -21,10 +21,23 @@
     @Steps
     ATFDetailsSteps atfDetailsSteps;
 
-    @Ignore("ignore until ticket is merged in develop")
-    @Title("CVSB - 1865 - AC1 - Sites list + AC2 - No search results")
-    public void testSitesList() {
+    @Steps
+    SiteVisitSteps siteVisitSteps;
+
+    @Steps
+    IdentifyVehicleSteps identifyVehicleSteps;
+
+    @Steps
+    VehicleDetailsSteps vehicleDetailsSteps;
+
+    @Steps
+    TestHistorySteps testHistorySteps;
 
+    @Steps
+    TestHistoryDetailsSteps testHistoryDetailsSteps;
+
+    @Title("CVSB - 1865 - AC1 - Sites list + AC2 - No search results")
+    @Test public void testSitesList(){
```

Comment:
We should have the @Test tag and the methode declaration on different lines

Reply:
Will update that now, thank you Dragos